vulture/core.py

```diff
@@ -130,6 +130,14 @@ def get_report(self, add_size=False):
             utils.format_path(self.filename), self.first_lineno,
             self.message, self.confidence, size_report)
 
+    def get_whitelist_string(self):
+        if self.typ != 'unreachable_code':
```

Comment:
It's usually simpler to understand if we use a positive test in the if-statement, i.e., `if self.typ == 'unreachable_code'`.